pkg/platform/kube/functionres/lazy.go

```diff
@@ -510,7 +511,7 @@ func (lc *lazyClient) deleteRemovedCronTriggersCronJob(functionLabels labels.Set
 	errGroup := errgroup.Group{}
```

Comment:
update the way errgroup is being created (e.g.: `errGroup, _ := errgroup.WithContext(ctx, lc.logger)`)